librz/bin/format/dex/dex.c

```diff
@@ -1564,12 +1560,12 @@ RZ_API RZ_OWN RzList /*<char *>*/ *rz_bin_dex_libraries(RZ_NONNULL RzBinDex *dex
 			object = p;
 		}
 
-		if (rz_list_find(libraries, object, compare_strings)) {
+		if (rz_pvector_contains(libraries, object)) {
```

Comment:
does this work well? I think this just compares the pointers, but the old behaviour compares the content, so this may creates problems.

Reply:
yeah, this is wrong

Reply:
My fault. Will revise it later. Thanks.

Reply:
don't forget to rename also the rz_vector_contains in the same way

Reply:
Decide to use `rz_pvector_find` instead of `rz_pvector_contains` for comparator-based searching considering there is also `rz_list_contains` performing the similar semantic as the current `rz_pvector_contains`. What we want for `rz_pvector_contains` is actually similar to `rz_list_find`. What do you think?

Reply:
sounds perfect.